src/app/components/home/home.component.html

```diff
@@ -0,0 +1,112 @@
+<section id="shopByCategoryGrid" class="my-lg-14 my-8">
```

Comment:
Do you need this id?

Reply:
it's easier for me to find myself in the code but it is not used otherwise